src/js/SubmissionContainer.jsx

```diff
@@ -57,11 +60,18 @@ var SubmissionContainer = React.createClass({
 
     if(code > 5) editsContainer = <EditsContainer/>
 
-    if(code > 8){
+    if(code > 9){
+      irs = <IRS/>;
+      uploadForm = null;
```

Comment:
Hmm, ok. I think I see what you mean. Basically just leave all the components there once they are added? And let them render however they'll render ... could be empty even but that's ok.